man/org.freedesktop.oom1.xml

```diff
@@ -56,12 +59,19 @@ node /org/freedesktop/oom1 {
 
     <variablelist class="dbus-method" generated="True" extra-ref="DumpByFileDescriptor()"/>
 
+    <variablelist class="dbus-signal" generated="True" extra-ref="Killed"/>
+
     <!--End of Autogenerated section-->
 
     <refsect2>
       <title>Methods</title>
 
       <para>...</para>
```

Comment:
Please drop this paragraph.

Reply:
Not addressed? I mean, please drop `<para>...</para>` entirely.

Reply:
Done. Sorry for dropping the wrong paragraph